opentracing/span.py

```diff
@@ -43,25 +43,6 @@ class Span(object):
     def __init__(self, tracer):
         self._tracer = tracer
 
-    def __enter__(self):
```

Comment:
https://github.com/uber-common/opentracing-python-instrumentation depends on Span implementing context manager. Why do you want to delete these?